packages/big-design/src/components/TableNext/mixins/display/display.tsx

```diff
@@ -0,0 +1,56 @@
+import { Breakpoints, breakpointsOrder, ThemeInterface } from '@bigcommerce/big-design-theme';
+import { css, FlattenSimpleInterpolation } from 'styled-components';
+
+import { TableColumnDisplayOverload, TableColumnDisplayProps } from './types';
+
+export const withTableColumnDisplay = () => css<TableColumnDisplayProps>`
+  ${({ display, theme }) => display && getDisplayStyles(display, theme, 'display')};
+`;
+
+const getDisplayStyles: TableColumnDisplayOverload = (
+  displayProp: any,
+  theme: ThemeInterface,
+  cssKey: any,
+): FlattenSimpleInterpolation => {
+  if (typeof displayProp === 'object') {
+    return getResponsiveDisplay(displayProp, theme, cssKey);
+  }
+
+  if (typeof displayProp === 'string' || typeof displayProp === 'number') {
+    return getSimpleDisplay(displayProp, cssKey);
+  }
+
+  return [];
+};
+
+const getSimpleDisplay = (
+  displayProp: string | number,
+  cssKey: string,
+): FlattenSimpleInterpolation => css`
+  ${cssKey}: ${displayProp}
+`;
+
+const getResponsiveDisplay: TableColumnDisplayOverload = (
```

Comment:
Wondering why we are not using the mixins that already exist in `src/mixins/display`?

Reply:
Not sure tbh, this was a copy paste from the Table component, it was like that before. But I'll take a look and see what I can do

Reply:
I already checked and I think is like that because the types are different, so I imagine that even-though it's almost the same, it was better to keep it separately to make it more readable ? 🤔  I think I'm gonna leave it like that and we can create another ticket to handle that scope.